src/main/java/com/mraof/minestuck/item/loot/GristLootModifier.java

```diff
@@ -0,0 +1,71 @@
+package com.mraof.minestuck.item.loot;
+
+import com.google.gson.JsonObject;
+import com.mraof.minestuck.alchemy.GristCostRecipe;
+import com.mraof.minestuck.alchemy.GristSet;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.util.GsonHelper;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.level.storage.loot.LootContext;
+import net.minecraft.world.level.storage.loot.parameters.LootContextParams;
+import net.minecraft.world.level.storage.loot.predicates.LootItemCondition;
+import net.minecraft.world.phys.Vec3;
+import net.minecraftforge.common.loot.GlobalLootModifierSerializer;
+import net.minecraftforge.common.loot.LootModifier;
+import org.apache.commons.compress.utils.Lists;
+
+import javax.annotation.Nonnull;
+import java.util.List;
+
+public class GristLootModifier extends LootModifier
+{
```

Comment:
Could you add a small bit of Java Doc? If I came across this I am not sure I would immediately understand what it is used for